index.js

```diff
@@ -220,6 +220,15 @@ function onStream (opts, client, onerror) {
       next()
     })
 
+    if (opts.payloadLogFile) {
+      if (!client._payloadLogFile) {
+        client._payloadLogFile = require('fs').createWriteStream(opts.payloadLogFile, {flags: 'a'})
```

Comment:
@Qard This will leave the file open forever. We could also just close it every time `stream` closed and then just reopen it again at the next request. Is there a downside of just leaving it open like this?

Reply:
Since it's intended just for debugging purposes, I don't think it matters much. I think it's fine to leave it as-is.